config/rubygems.yml

```diff
@@ -42,6 +46,7 @@ production:
 oidc-api-token:
   protocol: https
   host: oidc-api-token.rubygems.org
+  host_display: RubyGems.org oidc-api-token
```

Comment:
This env could be removed per my understanding.

Reply:
Done.